chainer/functions/pooling/average_pooling_nd.py

```diff
@@ -158,6 +217,11 @@ def average_pooling_nd(x, ksize, stride=None, pad=0):
             window size.
         pad (int or tuple of ints): Spatial padding width for the input array.
             ``pad=p`` and ``pad=(p, p, ..., p)`` are equivalent.
+        pad_value (0 or None):
+            Value to fill the padded region when calculating average.
+            If ``None`` is specified, such region is ignored.
+            The default value is ``0``, therefore the averages are biased
+            towards zero.
```

Comment:
How about adding notes similar to this to the docstring of `average_pooling_2d`?
(e.g., `Currently average_pooling_2d treats padded regions as filled by 0 when calculating average.`)

Reply:
Added note.